plugins/image-prioritizer/class-image-prioritizer-tag-visitor.php

```diff
@@ -36,4 +38,24 @@ abstract public function __invoke( OD_Tag_Visitor_Context $context ): bool;
 	protected function is_data_url( string $url ): bool {
 		return str_starts_with( strtolower( $url ), 'data:' );
 	}
+
+	/**
+	 * Gets attribute value for select attributes.
+	 *
+	 * @since n.e.x.t
+	 * @todo Move this into the OD_HTML_Tag_Processor/OD_HTML_Processor class eventually.
+	 *
+	 * @phpstan-param NormalizedAttributeNames $attribute_name
+	 *
+	 * @param OD_HTML_Tag_Processor $processor      Processor.
+	 * @param string                $attribute_name Attribute name.
+	 * @return string|true|null Normalized attribute value.
+	 */
+	protected function get_attribute_value( OD_HTML_Tag_Processor $processor, string $attribute_name ) {
```

Comment:
I'd like to move this eventually into `OD_HTML_Tag_Processor`/`OD_HTML_Processor`. Also @dmsnell has suggested this could eventually be part of the core class as well, as most HTML attributes have their own dedicated logic to get back a normalized value.